HUD/SetMinimapComponent.md

```diff
@@ -9,14 +9,36 @@ Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
 ```
 
 This native is used to colorize certain map components like the army base at the top of the map.
+Colors are being mixed, so for example using blue (HudColor: 9) on yellow lift-off markers
+will result in green instead of blue, because the base color is yellow.
 
-An incomplete list of components ID:
+**A complete list of components ID:**
+
+
+0: Los Santos' air port yellow lift-off markers. <br>
```

Comment:
`<br>` isn't needed and wont get formatted properly, instead you should use a list
```
**Some Title**

* Some Element
* Another Element